config/pod.yaml.hbs

```diff
@@ -30,6 +30,8 @@ spec:
         value: {{runtimeClass}}
       - name: SD_PUSHGATEWAY_URL
         value: {{pushgateway_url}}
+      - name: SD_TERMINATION_GRACE_PERIOD_SECONDS
```

Comment:
using this to control wait in run.sh as a workaround to wait for launcher to execute abort steps. as discussed with @pritamstyz4ever we will revisit this